### PR TITLE
Update banner width

### LINKED ISSUE #494: Image Banner - Content Max Width
Provide additional flexibility for merchants to be able to have their text content on image banner occupy more width. 

When content is more verbose, the current mad width for the content feels restrained on desktop. 

image

Perhaps we revisit the max width, or perhaps we provide for different width options.

### PR DESCRIPTION
**Pull request by sofiamatulis**
**Why are these changes introduced?**

Fixes #494 .

**What approach did you take?**

Update the max and min width values

**Other considerations**

**Demo links**

- [Editor](https://os2-demo.myshopify.com/admin/themes/126418419734/editor)

**Checklist**
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by kmeleta — DISMISSED** [review]
Code looks good and I think the widths chosen work well 👍

File: assets/section-image-banner.css [hunk 1]
```diff
@@ -181,12 +184,13 @@
     --color-foreground: 255, 255, 255;
     --color-button: 255, 255, 255;
     --color-button-text: 0, 0, 0;
+    max-width: 89.5rem;
```
Comment (melissaperreault) [inline assets/section-image-banner.css:187]:
```suggestion
    max-width: 89rem;
```
Cleaning up some values, the 0.5 didn't make a difference here.

**Review by melissaperreault — DISMISSED** [review]
Thank you so much Sofia! Looking good! Left small comments to clean up some values but overall ready to go!

I was afraid to need to add additional breakpoint number of lines of code but ends up that the `banner_box` padding adds enough safe space to have a good balance when the title becomes longer.

**Review by LucasLacerdaUX — APPROVED** [review]
Looks good! :DD

**Review by kmeleta — APPROVED** [review]
:ship:

**Review by melissaperreault — APPROVED** [review]

**Pull request merged**